src/mui/detail/Edit.js

```diff
@@ -64,10 +72,11 @@ export class Edit extends Component {
 
     render() {
         const { actions = <DefaultActions />, children, data, hasDelete, hasShow, id, isLoading, resource, title } = this.props;
+        const { key } = this.state;
         const basePath = this.getBasePath();
 
         return (
-            <Card style={{ margin: '2em', opacity: isLoading ? 0.8 : 1 }}>
+            <Card style={{ margin: '2em', opacity: isLoading ? 0.8 : 1 }} key={key}>
```

Comment:
depends on `isLoading`, so it's dynamic. It could be optimized, but that's not the purpose of that PR. Same for your other comment.